### PR TITLE
Feat/Playbook/Add_Hash_VT

### PR DESCRIPTION
**Pull request by Thibault-MAL**
Hello SEKOIA.IO

For today, I propose you a new playbook, using the best of VirusTotal, Hash scan.
(sha1 for being accurate)

Best regards

### CODE REVIEW

File: playbooks/templates/Enrich_alerts_with_VirusTotal_Hash.json [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+      "name": "Comment Alert",
+      "type": "action",
+      "outputs": {
+        "default": []
+      },
+      "arguments": {
+        "uuid": "{{ node.0['alert_uuid'] }}",
+        "content": "This file.hash.sha1: \"{{ node.12['output'][0] }}\" is not know to be malicious."
```
Comment (GuillaumeCSekoia) [inline playbooks/templates/Enrich_alerts_with_VirusTotal_Hash.json:111]:
```suggestion
        "content": "This file.hash.sha1: \"{{ node.12['output'][0] }}\" is not known to be malicious."
```

File: playbooks/templates/Enrich_alerts_with_VirusTotal_Hash.json [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+      },
+      "action_uuid": "0d323de3-a864-4afe-a0c3-e7ff45883d7a",
+      "module_uuid": "92d8bb47-7c51-445d-81de-ae04edbb6f0a"
+    }
+  },
+  "workspace": "Operation Center",
+  "description": "Enrich with VirusTotal to check if this file.hash.sha1 is known from this service direclty from SEKOIA.IO."
```
Comment (GuillaumeCSekoia) [inline playbooks/templates/Enrich_alerts_with_VirusTotal_Hash.json:118]:
```suggestion
  "description": "Enrich to check if this file.hash.sha1 is known from VirusTotal, directly from SEKOIA.IO."
```

File: playbooks/templates/playbooks.json [hunk 3]
```diff
@@ -50,14 +50,21 @@
...
             "alert",
             "enrichement"
         ],
         "workspace": ["Operation Center"]
     },
+    {
+        "file": "Enrich_alerts_with_VirusTotal_Hash.json",
+        "name": "Scan for hash on VirusTotal",
+        "workspace": ["Operation Center"],
+        "description": "Enrich with VirusTotal to check if this file.hash.sha1 is known from this service direclty from SEKOIA.IO.",
```
Comment (GuillaumeCSekoia) [inline playbooks/templates/playbooks.json:65]:
```suggestion
        "description": "Enrich to check if this file.hash.sha1 is known from VirusTotal, directly from SEKOIA.IO.",
```

**Review by GuillaumeCSekoia — DISMISSED** [review]
Did not look at the code but some typos in the text zones

**Review by PierrickV — APPROVED** [review]

**Pull request merged**